karapace/schema_backup.py

```diff
@@ -200,7 +204,7 @@ def parse_args():
     parser_restore = subparsers.add_parser("restore", help="Restore the schema backup from a file")
     for p in {parser_get, parser_restore}:
         p.add_argument("--config", help="Configuration file path", required=True)
-        p.add_argument("--location", help="File path for the backup file", required=True)
+        p.add_argument("--location", default=None, help="File path for the backup file")
```

Comment:
For restore this should probably still be required.

Reply:
We're doing a check for restore, asserting that the location actually exists. I'll just update the default value to empty string, ensuring that check will fail with the proper error message